### PR TITLE
#12 - add link to install/open provider's app

### LINKED ISSUE #12: Add link to install/open provider's application
When user is on the provider's page, he should some link/button to open application or install it.

### PR DESCRIPTION
**Pull request by zmigrOO**
```>php artisan migrate:fresh --seed``` is required for this to work.
Backend works fine, only frontend is to be approved. 
It should close #12

### CODE REVIEW

**Review by AleksandraKozubal — COMMENTED** [review]
Please add "It should close..." and screens of frontend changes

**Comment by zmigrOO** [comment]
[Preview of working buttons](https://github.com/blumilksoftware/escooters/assets/100092805/7cc12e0a-bcb1-4e06-83a9-efc00ab5f9b6)

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. Funfact - when branch's name has slash, Phpstorm shows it like that

image

I think that name of branch should be simple, like `#12-add-link-to-open-install-providers-app`

2. I'm not a big fan of this list. Maybe darker buttons (like `bg-blumilk-500`) will look better? Or maybe white background? I don't know

image

File: resources/js/Shared/Components/ProviderIcons.vue [hunk 1]
```diff
@@ -10,16 +12,86 @@ function getProviderColor(providerName) {
...
         <img loading="lazy" class="w-7 lg:w-8" :src="'/providers/' + cityProvider.provider_name.toLowerCase() + '.png'" alt="">
       </div>
     </div>
+    <div class="provider-buttons border border-solid border-blumilk-100 bg-white shadow-lg">
+      <p class="text-center text-lg">
+        {{ urls.pname }}
+      </p>
+      <a v-if="urls.url" :href="urls.url" target="_blank" class="flex h-11 w-36 flex-row place-items-center justify-items-center rounded bg-blumilk-300 shadow-inner">
+        <img loading="lazy" class="mx-2 w-6" src="/icons/globe.svg" alt="">
+        <p class="provider-button text-lg  font-semibold text-white hover:underline">Web</p>
```
Comment (AleksandraKozubal) [inline resources/js/Shared/Components/ProviderIcons.vue:80]: What is the `provider-button` class used for?

File: resources/js/Shared/Components/ProviderIcons.vue [hunk 2]
```diff
@@ -10,16 +12,86 @@ function getProviderColor(providerName) {
...
+  urls.pname = provider.name
+  urls.url = provider.url ? provider.url : null
+  urls.android_url =  provider.android_url ? provider.android_url : null
+  urls.ios_url = provider.ios_url ? provider.ios_url : null
+  console.log(provider)
+} 
+
+if (window.location.pathname !== '/') {
+  document.addEventListener('click', function (event) {
+    // close provider buttons if clicked outside
```
Comment (AleksandraKozubal) [inline resources/js/Shared/Components/ProviderIcons.vue:26]: Please, remove those three comments

**Review by AleksandraKozubal — COMMENTED** [review]
My proposal is to make this window sth like that:
1. provider's name color blumilk-500 and enlarge pb to 2px
2. buttons text with font-weight-normal and button's color to blumilk-400
3. remove blumilk color from window's border

image

**Comment by zmigrOO** [comment]
> My proposal is to make this window sth like that:
> 1. provider's name color blumilk-500 and enlarge pb to 2px
> 2. buttons text with font-weight-normal and button's color to blumilk-400
> 3. remove blumilk color from window's border
> 
> Probably rounding the corners of buttons would also look nicer
> 
> image
> 

You mean to round the corners more?

**Review by AleksandraKozubal — APPROVED** [review]

**Review by vojcc — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**